examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
ohh ok, still can't figure out why it keeps erroring with "Unsupportes protocol"...